SCClassLibrary/Common/UnitTesting/UnitTest.sc

```diff
@@ -67,6 +88,8 @@ UnitTest {
 	}
 
 	// run a single test method of this class
+	// this assumes that setUpClass has been called
+	// and that tearDownClass is called afterwards
```

Comment:
should also be in documentation, or may we should make `runTestMethod` private?

Reply:
we may want to do this, yes. Not in the current PR though, let's do another one for this.

Reply:
sure! sorry, i didn't mean to imply that should happen in this PR.